.github/workflows/pipeline-deploy-app-test.yaml

```diff
@@ -111,35 +111,16 @@ jobs:
           export DEPLOYMENT_WORKSPACE=${{ needs.derive-deployment-workspace.outputs.deployment_workspace }}
           /bin/bash ./scripts/app-build-deploy.sh
   integration-test:
-    name: Run integration tests
-    runs-on: ubuntu-latest
-    needs:
-      [
-        get-metadata,
-        deploy-application-infrastructure,
-        build-and-deploy-application-code,
-        derive-deployment-workspace,
-      ]
-    permissions:
-      id-token: write
-      contents: read
-    environment: "test"
-    timeout-minutes: 10
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          ref: ${{ inputs.git_tag }}
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v2.2.0
-        with:
-          role-to-assume: arn:aws:iam::${{ secrets.ACCOUNT_ID }}:role/uec-dos-service-management-github-runner
-          role-session-name: github-pipeline-session
-          aws-region: ${{ vars.AWS_REGION }}
-      - name: Run Integration tests
-        run: |
-          export BUILD_DATETIME="${{ needs.get-metadata.outputs.build_datetime }}"
-          export BUILD_TIMESTAMP="${{ needs.get-metadata.outputs.build_timestamp }}"
-          export BUILD_EPOCH="${{ needs.get-metadata.outputs.build_epoch }}"
-          export DEPLOYMENT_WORKSPACE=${{ needs.derive-deployment-workspace.outputs.deployment_workspace }}
-          echo "Integration tests invoked"
-          /bin/bash ./scripts/integration-tests.sh
+      needs:
+        [
+          deploy-application-infrastructure,
+          build-and-deploy-application-code,
+          derive-deployment-workspace,
+        ]
+      name: Run integration tests
+      uses: ./.github/workflows/integration-test.yaml
+      with:
+        github_environment: test
```

Comment:
See above ... is this used in callled workflow ?

Reply:
Isn't this required to by action-stack?